guides/common/modules/proc_creating-a-custom-ssl-certificate.adoc

```diff
@@ -74,8 +74,8 @@ commonName = _{ssl-common-name}_ <1>
 
 [ v3_req ]
 basicConstraints = CA:FALSE
-keyUsage = digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment
-extendedKeyUsage = serverAuth, clientAuth, codeSigning, emailProtection
+keyUsage = digitalSignature, keyEncipherment
+extendedKeyUsage = serverAuth, clientAuth
```

Comment:
The testing of this uses this configuration which includes some of these (as it was following the docs): https://github.com/SatelliteQE/robottelo/blob/7306155b578f1f574f7ad07060b09fdfaa8d9e3f/tests/foreman/data/openssl.cnf#L60-L61

We should update that to match this.